code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -66,6 +66,14 @@ Credit where due:
 
 /proc/add_servant_of_ratvar(mob/living/L, silent = FALSE)
 	var/update_type = /datum/antagonist/clockcultist
+	if(iscyborg(L))
+		var/mob/living/silicon/robot/R = L
+		if(R.deployed)
+			var/mob/living/silicon/ai/ai = R.mainframe
+			R.undeploy()
+			ai << "<span class='heavy_brass'>Error. Surge Detected. Returning to core..</span>"
+			add_servant_of_ratvar(ai, FALSE)
+			return TRUE
```

Comment:
This code should NOT be in this proc; instead it should be in `give_to_body()`

    /datum/antagonist/clockcultist/give_to_body(mob/living/new_body)
    	if(iscyborg(new_body))
    		var/mob/living/silicon/robot/R = new_body
    		if(R.deployed)
    			var/mob/living/silicon/ai/AI = R.mainframe
    			R.undeploy()
    			var/converted = add_servant_of_ratvar(AI, silent_update)
    			AI << "<span class='userdanger'>You have been forced to return to your core!</span>"
    			return converted
    	[rest of code here]